clustered/common/src/main/java/org/ehcache/clustered/common/messages/ServerStoreOpMessage.java

```diff
@@ -53,16 +54,34 @@ public static ServerStoreOp getServerStoreOp(byte storeOpCode) {
           throw new IllegalArgumentException("Store operation not defined for : " + storeOpCode);
       }
     }
+
   }
 
+  private static final Charset UTF_8 = Charset.forName("UTF-8");
```

Comment:
The encode/decode operations based on this are likely to be a lot faster if you simple transfer the `char[]`from the String.  Using the UTF-8 `Charset` is _expensive_ and subject to some _unannounced_ data modification if the String happens to have some "bad" characters.